age/keysource_test.go

```diff
@@ -0,0 +1,31 @@
+package age
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAge(t *testing.T) {
+	assert := assert.New(t)
+
+	key, err := MasterKeyFromRecipient("age1yt3tfqlfrwdwx0z0ynwplcr6qxcxfaqycuprpmy89nr83ltx74tqdpszlw")
+
+	assert.NoError(err)
+	assert.Equal("age1yt3tfqlfrwdwx0z0ynwplcr6qxcxfaqycuprpmy89nr83ltx74tqdpszlw", key.ToString())
+
+	dataKey := []byte("abcdefghijklmnopqrstuvwxyz123456")
+
+	err = key.Encrypt(dataKey)
+	assert.NoError(err)
+
+	pwd, err := os.Getwd()
+	assert.NoError(err)
+	err = os.Setenv("SOPS_AGE_KEY_DIR", pwd)
```

Comment:
You cannot rely always on the working directory here:

```suggestion
	_, filename, _, _ := runtime.Caller(0)
	err = os.Setenv("SOPS_AGE_KEY_DIR", path.Dir(filename))
```

Reply:
Since `SOPS_AGE_KEY_DIR` is set to a constant string this should be fine as long as all tests are relying on the same key source directory.